Tests/XCUITests/HomePageSettingsUITest.swift

```diff
@@ -220,6 +220,30 @@ class HomePageSettingsUITests: BaseTestCase {
 //        waitForExistence(app.cells.staticTexts["Example Domain"], timeout: 3)
     }
 
+    func testRecentlyVisited() {
+        navigator.openURL(websiteUrl1)
+        waitUntilPageLoad()
+        navigator.performAction(Action.GoToHomePage)
+        waitForExistence(app.scrollViews.cells[AccessibilityIdentifiers.FirefoxHomepage.HistoryHighlights.itemCell].staticTexts["Internet for people, not profit — Mozilla"])
```

Comment:
Nit: can you please add a variable for the staticTexts label? it is used in several places so better to have just one place to modify in case it changes... 
Thanks!